roles/pulp/tasks/main.yaml

```diff
@@ -138,8 +132,20 @@
     command: pulpcore-manager migrate --noinput
     network: host
     secrets:
-      - 'pulp-settings-py,type=mount,target=/etc/pulp/settings.py'
       - 'pulp-symmetric-key,type=mount,target=/etc/pulp/certs/database_fields.symmetric.key'
+    env:
+      PULP_ANALYTICS: "{{ pulp_enable_analytics | ternary('true', 'false') }}"
+      PULP_AUTHENTICATION_BACKENDS: "['pulpcore.app.authentication.PulpNoCreateRemoteUserBackend']"
+      PULP_CACHE_ENABLED: "true"
+      PULP_CONTENT_ORIGIN: "{{ pulp_content_origin }}"
+      PULP_REDIS_URL: "redis://localhost:6379/8"
+      PULP_REMOTE_USER_ENVIRON_NAME: "HTTP_REMOTE_USER"
+      PULP_REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES: "['rest_framework.authentication.SessionAuthentication', 'pulpcore.app.authentication.PulpRemoteUserAuthentication']"
+      PULP_DATABASES__default__HOST: "localhost"
+      PULP_DATABASES__default__NAME: "pulp"
+      PULP_DATABASES__default__PASSWORD: "{{ pulp_db_password }}"
```

Comment:
I think with normal systemd services the env vars are world readable. Is that also the case here?

Reply:
yes, there are reasons this PR is still a draft ;-)